src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -130,6 +152,22 @@ define('Core/Commander/Interfaces/ApiInterface/ApiGlobe', [
         return false;
     };
 
+
+    /**
+    * Add an elevation layer to the map. Elevations layers are used to build the terrain, if there is some overlapped the best resolution is taken, if resolution is equals, the first one is used.
```

Comment:
"if there is some overlap the best resolution is taken, if the resolutions are equal, the first one is used"